SPM Example/PortalSwift/ViewController.swift

```diff
@@ -565,24 +565,24 @@ class ViewController: UIViewController, UITextFieldDelegate {
       throw PortalExampleAppError.portalNotInitialized()
     }
     let chainId = "eip155:11155111"
```

Comment:
we should have a method for sending eth that uses build transaction as well as builds it ourselves